dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemUploadControllerIT.java

```diff
@@ -0,0 +1,538 @@
+        context.restoreAuthSystemState();
+        getClient(token).perform(MockMvcRequestBuilders.fileUpload("/api/core/items/" + item.getID() + "/bitstreams")
+                                                       .file(file).file(file2))
+                        .andExpect(status().isUnprocessableEntity());
+    }
+
+    @Test
+    public void uploadBitstreamNoBundleNameInPropertiesUnprocessableException() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item item = ItemBuilder.createItem(context, col1)
+                               .withTitle("Author1")
+                               .withIssueDate("2017-10-17")
+                               .withAuthor("Smith, Donald")
+                               .build();
+        context.setCurrentUser(eperson);
+        authorizeService.addPolicy(context, item, Constants.ADD, eperson);
+        authorizeService.addPolicy(context, item, Constants.WRITE, eperson);
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        String input = "Hello, World!";
+
+        MockMultipartFile file = new MockMultipartFile("file", "hello.txt", MediaType.TEXT_PLAIN_VALUE,
+                                                       input.getBytes());
+
+        BitstreamPropertiesRest bitstreamPropertiesRest = new BitstreamPropertiesRest();
+
+        ObjectMapper mapper = new ObjectMapper();
+
+
+        context.restoreAuthSystemState();
+        getClient(token).perform(MockMvcRequestBuilders.fileUpload("/api/core/items/" + item.getID() + "/bitstreams")
+                                                       .file(file)
+                                                       .param("properties", mapper
+                                                           .writeValueAsString(bitstreamPropertiesRest)))
+                        .andExpect(status().isUnprocessableEntity());
+
+        getClient(token).perform(get("/api/core/items/" + item.getID() + "/bitstreams"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("_embedded.bitstreams").doesNotExist());
+    }
+
+
+    //TODO This check on sequence ID doesn't happen
```

Comment:
I do not understand this TODO comment.  Could you clarify?

Reply:
This was an old comment from during the implementation. It was fixed prior to the creation of the PR. The comment has been removed since the IT works correctly